.github/workflows/prettier.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettier Markdown
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'docs/**'
+
+jobs:
+  pretty:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+      - run: npm install -g yarn
```

Comment:
`yarn` should already be installed as part of `setup-node`, unless it was removed in v2

Reply:
was using ACT, finding it is not full compatible with GA, have removed the install yarn line